scipy-1.0/paper.tex

```diff
@@ -218,6 +222,55 @@ \section*{Discussion}
 \textbf{Future development}.
 \textit{This section should include key issues: sparse arrays, ndimage pixel vs point, splines, fftpack vs. np.fft and linalg vs. np.linalg, under-maintained submodules.}
 
+\subsection{ndimage pixel vs point}
+
+\begin{itemize}
+  \item A pixel is not a little square
+    (http://alvyray.com/Memos/CG/Microsoft/6\_pixel.pdf)
+  \item Match edges vs match centers
+    (http://entropymine.com/imageworsener/matching/)
+\end{itemize}
+
+There are two competing models for representing image measurements:
+the first considers an image as a collection of pixels, square blocks
+of size 1x1 with centers offset by (0.5, 0.5).  This model corresponds
+closely with a camera sensor, that integrates light as it falls on
+(more-or-less) square elements, producing a digital readout for each.
+
+Another view is that of an image as a grid of point measurements.
+This model explicitly reject two notions encouraged by the pixel
+model: that our signal is best represented as a step-wise functio, and
+that a single measurement represents knowledge of the surrounding
+square area element.  Instead, we think of the image as a smoothly
+varying function, of which we have measurements at grid positions.
+
+While the representations may not seem very different, they have
+significant implications for image processing algorithms such as
+rescaling.
+
+{\tt ndimage} currently follows the pixel model, but we think that the
+data point model will serve us better in the long run:
+
+\begin{enumerate}
+  \item It makes no assumption about the sampling mechanism used
+    (camera, scanner, laser, etc.).
+  \item NumPy coordinates will map directly to image coordinates,
+    without a 0.5 offset.
+  \item It makes it easier to reason about extrapolation.  E.g., the current
+    dubious notion of ``reflection'', in which the image is padded
+    {\tt d c b a | a b c d | d c b a} becomes nonsensical, and leaves
+    only the reliable notion of mirroring, {\tt d c b | a b c d | c b a}.
+\end{enumerate}
+
+The second model, while consistent, does introduce some challenges
+w.r.t. traditional image manipulation (see attached figure).
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.5\textwidth]{static/image_pixel_model}
+    \caption{Caption goes here.}
```

Comment:
A short caption would be helpful before merging since this stuff requires some domain expertise (or maybe refer me to the appropriate reading material I guess).